cypress/tests/data/60-content/VkarbasizaedSubmission.spec.js

```diff
@@ -53,10 +57,151 @@ describe('Data suite tests', function() {
 		cy.get('button').contains('Continue').click();
 		cy.get('button').contains('Continue').click();
 		cy.get('button').contains('Complete').click();
+	});
+
+	it('Schedule for publication', function() {
+		cy.login('dbarnes');
+		// schedule for the publication in the future issue
+		cy.visit('index.php/publicknowledge/submissions');
+		cy.get('button[id="active-button"]').click();
+		cy.contains('View ' + familyName).click({force: true});
+		cy.get('button[id="publication-button"]').click();
+		cy.get('div#publication button:contains("Schedule For Publication")').click();
+		cy.get('select[id="assignToIssue-issueId-control"]').select(issueTitle);
+		cy.get('div[id^="assign-"] button:contains("Save")').click();
+		cy.get('div:contains("All publication requirements have been met. This will be published when ' + issueTitle + ' is published. Are you sure you want to schedule this for publication?")');
+		cy.get('div.pkpWorkflow__publishModal button:contains("Schedule For Publication")').click();
+		// check status = 5 (scheduled)
+		cy.reload() // to be able to get the header
+		// check submission status
+		cy.get('span').should('have.class', 'pkpWorkflow__identificationStatus').contains("Scheduled");
+		// check publication status
+		cy.get('button[id="publication-button"]').click();
+		cy.get('span').should('have.class', 'pkpPublication__status').contains("Scheduled");
+		// the button "Unschedule" exists
+		// the buttons "Create New Version" (connected with submission) and "Unpublish" (connected with publication) does not exist
+		cy.get('div#publication button:contains("Unschedule")');
+		cy.get('div#publication button:contains("Create New Version")').should('not.exist');
+		cy.get('div#publication button:contains("Unpublish")').should('not.exist');
```

Comment:
It should be possible to to check `cy.get('span').should('have.class', 'pkpWorkflow__identificationStatus').contains("Scheduled");` without a page reload and replace all this code with the one check. You may just need to add a short delay for the request to return. I recommend trying `1000` at first.

You can also write your checks to be a little less tied to exact elements. That will make them more resilient if there are minor changes to class names or HTML structure. For example, instead of:

```
cy.get('span').should('have.class', 'pkpWorkflow__identificationStatus').contains("Scheduled");
```

I might write:

```
cy.get('.pkpWorkflow__header:contains("Scheduled")');
```